src/Umbraco.AuthorizedServices/Configuration/AuthorizedServiceSettings.cs

```diff
@@ -253,6 +253,16 @@ public class ServiceDetail : ServiceSummary
     /// </summary>
     public string? SampleRequest { get; set; }
 
+    /// <summary>
+    /// Gets or sets the time interval for expiration of access tokens.
+    /// </summary>
+    public TimeSpan AccessTokenExpirationInterval { get; set; } = TimeSpan.FromSeconds(30);
```

Comment:
Would suggest naming here may be clearer as: `RefreshAccessTokenWhenExpiresWithin` and `ExchangeTokenWhenExpiresWithin`.  And perhaps move the latter into the `ExchangeTokenProvision` structure.